package.json

```diff
@@ -21,7 +21,7 @@
         "build": "turbo build",
         "build:demo": "turbo build:demo",
         "build:storybook": "storybook build",
-        "lint": "eslint .",
+        "lint": "eslint . --quiet",
```

Comment:
If you just want to show Error messages, you can run `pnpm lint - -quiet`, if `-quiet` is preset here, you won't be able to view other non-Error messages